pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1476,7 +1476,7 @@ def test_tdi_rmul_arraylike(self, other, box_with_array):
 
         tdi = TimedeltaIndex(['1 Day'] * 10)
         expected = timedelta_range('1 days', '10 days')
-        expected._data._freq = None
+        expected._freq = None
```

Comment:
this was one of the places where expected.freq didnt match expected._data.freq

Reply:
To be clear,  we want `expected._data._freq = None` here, right? Setting `expected._freq` shouldn't do anything. The index classes should always have a reference to the array's freq.

Reply:
Definitely.  The change implemented here should be made unnecessary.